frontend/src/global_styles/content/_widget_box.sass

```diff
@@ -105,9 +105,8 @@ $widget-box--enumeration-width: 20px
 
     .widget-box--description
       display: grid
-      grid-template-columns: auto auto
```

Comment:
These changes also impact the "what's new" teaser. From a quick check I'd say that has no effect on the EE block overall, so it can probably be reverted?